### ISSUE TITLE
No prospect lists Relation in contacts module 

### ISSUE THREAD

**Issue by sanftleben** [SuiteCRM/SuiteCRM#5835]
You can add a contact to a prospect list in the subpanel of a prospect list. Yet, this is not visibile by default in contacts module. So you won't always know on what prospect list your contact is acutally on. But that's not the real issue. 

#### Issue
The issue is that when you merge two (or more) contacts, relations to the contact list are not being copied. So when you have duplicate contacts on different prospect lists, you're likely to lose track.

#### Expected Behavior
Prospect list relations should be mergen as well.

#### Actual Behavior
In modules/MergeRecords/SaveMerge.php Line 149 the relations to the source are loaded. Yet, there is no relationship (at least none in the studio) to the prospect list and module. So ids are not copied. 

#### Possible Fix
What I did is to remove the deletion of the target id in the prospect_lists_prospects table (yeah contact relations are stored there). Very dirty but solves the issue:

$sql = "UPDATE prospect_lists_prospects SET deleted=0 WHERE related_id='$return_id' AND related_type='$return_module'";

#### Steps to Reproduce
1. Merge one or more contact that's on a prospect list

#### Your Environment

* SuiteCRM Version used: 7.10.4
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Chrome (latest, 64bit)
* Environment name and version (e.g. MySQL, PHP 7): MySQL, PHP 7
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 16.04 LTS

**Comment by LuminisIndia** [comment]
@antivist You can practically add the custom subpanel 'Target List' via coding in contacts module.  I don't know if we can do something of this sort via Studio.

**Comment by antivist** [comment]
Yes I've done that but when merging contacts the relation was not copied by default.
